components/homeComponents/HackCountdown.tsx

```diff
@@ -0,0 +1,203 @@
+}
+
+const HackCountdown: React.FC<Hack24CountdownProps> = ({ targetDate }) => {
+  const calculateTimeLeft = () => {
+    const difference = +new Date(targetDate) - +new Date();
+    let timeLeft = {
+      days: 0,
+      hours: 0,
+      minutes: 0,
+    };
+
+    if (difference > 0) {
+      timeLeft = {
+        days: Math.floor(difference / (1000 * 60 * 60 * 24)),
+        hours: Math.floor((difference / (1000 * 60 * 60)) % 24),
+        minutes: Math.floor((difference / 1000 / 60) % 60),
+      };
+    }
+
+    return timeLeft;
+  };
+
+  const [timeLeft, setTimeLeft] = useState(calculateTimeLeft());
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setTimeLeft(calculateTimeLeft());
+    }, 1000);
+
+    return () => clearTimeout(timer);
+  });
+
+  const digitBoxStyle: CSSProperties = {
+    background: 'linear-gradient(to bottom, rgba(156, 166, 255, .4), rgba(255, 255, 255, 0.1))',
+    boxShadow: '0 4px 8px rgba(0, 0, 0, 0.1)',
+    borderRadius: '5px',
+    margin: '0 1vw',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: '10vw',
+    height: '15vw',
+    border: '.5px solid #05149C',
+  };
+
+  const digitStyle: CSSProperties = {
+    fontFamily: '"DS-Digital", monospace',
+    fontSize: '10vw',
+    fontWeight: 400,
+    color: '#05149C',
+  };
+
+  const labelStyle: CSSProperties = {
+    textTransform: 'uppercase',
+    fontFamily: 'Poppins',
+    fontWeight: 400,
+    fontSize: '4vw',
+    lineHeight: '5vw',
+    color: '#05149C',
+    marginTop: '1vw',
+  };
+
+  const styles: { [key: string]: CSSProperties } = {
+    countdownContainer: {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      flexDirection: 'column',
+      fontFamily: 'Arial, sans-serif',
+      color: '#2E0480',
+      background:
+        '#FFFFFF url("data:image/svg+xml,%3Csvg width%3D%2250vw%22 height%3D%22353px%22 viewBox%3D%220 0 100vw 706px%22 fill%3D%22none%22 xmlns%3D%22http%3A//www.w3.org/2000/svg%22%3E%3Cdefs%3E%3Cfilter id%3D%22blurStrong%22%3E%3CfeGaussianBlur stdDeviation%3D%2220%22/%3E%3C/filter%3E%3C/defs%3E%3Cellipse cx%3D%2286.5vw%22 cy%3D%22260px%22 rx%3D%22180px%22 ry%3D%22180px%22 fill%3D%22%234A3AFF%22 opacity%3D%220.1%22 filter%3D%22url(%23blurStrong)%22/%3E%3Cellipse cx%3D%2299vw%22 cy%3D%22250px%22 rx%3D%22160px%22 ry%3D%22160px%22 fill%3D%22%232D5BFF%22 opacity%3D%220.1%22 filter%3D%22url(%23blurStrong)%22/%3E%3C/svg%3E") no-repeat bottom right',
+      height: '100vh',
+      padding: '2vw',
+      position: 'relative',
+      overflow: 'hidden',
+    },
+    timeSection: {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      gap: '1.5vw',
+    },
+    notifySection: {
+      marginTop: '2vw',
+      textAlign: 'center',
+      fontFamily: 'Poppins',
+      fontWeight: 400,
+      fontSize: '2.5vw',
+      lineHeight: '3.5vw',
+      color: '#000000',
+      width: '90%',
+    },
+  };
```

Comment:
nit: any reason why we're using pure CSS here instead of tailwindcss?

Reply:
I found it a lot easier to program specific details with pure CSS. I can implement it in tailwindcss if you want me to.

Reply:
it's just nit so I think refactor later if needed to.